_data/sites.json

```diff
@@ -6594,6 +6594,16 @@
             "recargapay.workable.com"
         ]
     },
+    
+    {
+        "name": "peopleperhour",
+        "url": "https://www.peopleperhour.com/settings/general",
+        "difficulty": "medium",
+        "notes": "Profile > settings > account > edit > select a reason > deactivate",
```

Comment:
Do you know if it gets actually deleted or just deactivated? Because if it's the latter, then we usually say it's "impossible", as you can't erase your presence completely.

Reply:
When you select "please confirm you understand that:" it says "Your profile together with reviews and portfolio will be deleted and buyers will no longer be able to find and contact you".

Reply:
Ah, ok, thanks for the clarification.